HotelBooking/Views/checked_in_rooms_view.py

```diff
@@ -1,16 +1,21 @@
 from __future__ import annotations
+from HotelBooking.Controllers.bill_controller import BillController
+from HotelBooking.Views.reserved_rooms_view import ReservedRoomsView
 from HotelBooking.Views.view import View
 from typing import Tuple, List
 from PyInquirer import prompt
+from HotelBooking.Controllers.reservation_controller import ReservationController
 from HotelBooking.Controllers.room_controller import RoomController
-from HotelBooking.Models.room import ROOM_TYPE
+from HotelBooking.Models.room import ROOM_TYPE, Room
 
 PROMPT_KEY = {
     "OPERATIONS": "operations",
     "CHECK-IN": 'check-in',
     "ROOM-CHECK-IN": 'room-check-in',
     "CHECK-OUT": 'check-out',
-    "CUSTOMER": "customer"
+    "CUSTOMER": "customer",
+    "BACK": "back",
+    "LIST_RESERVATION": "listOutReservation"
```

Comment:
can you use consistent naming schemes? consider `LIST-RESERVATION`: `list-out-reservation`. Also, why is the key mapped to a value that is different?